packages/react-core/src/components/Icon/examples/IconProgress.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Icon, Spinner } from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+
+export const IconProgress: React.FunctionComponent = () => (
+  <React.Fragment>
+    <Icon isInProgress progressIcon={<Spinner size="md" isSVG aria-label="Loading..." />} containerSize="md">
+      <CheckCircleIcon />
+    </Icon>
+    <Icon progressIcon={<Spinner size="md" isSVG aria-label="Loading..." />} containerSize="md">
```

Comment:
I believe `containerSize` is redundant since `md` is the default

```suggestion
    <Icon progressIcon={<Spinner size="md" isSVG aria-label="Loading..." />}>
```